pootle/depcheck.py

```diff
@@ -1,183 +0,0 @@
-        else:
-            return False, __version__
-    except ImportError:
-        return None, None
-
-
-def test_redis_server_available():
-    from django_rq.queues import get_queue
-    from django_rq.workers import Worker
-    from redis.exceptions import ConnectionError
-    queue = get_queue()
-    try:
-        workers = Worker.all(queue.connection)
-        return True, queue.connection.connection_pool.connection_kwargs
-    except ConnectionError:
-        return False, queue.connection.connection_pool.connection_kwargs
-
-
-def test_redis_server_version():
-    from django_rq.queues import get_queue
-    queue = get_queue()
-    redis_version_string = queue.connection.info()['redis_version']
-    redis_version = tuple([int(x) for x in redis_version_string.split(".")])
-    if redis_version >= REDIS_MINIMUM_REQUIRED_SERVER_VERSION:
-        return True, redis_version_string
-    else:
-        return False, redis_version_string
-
-
-def test_rq_workers_running():
-    from django_rq.queues import get_queue
-    from django_rq.workers import Worker
-    queue = get_queue()
-    workers = Worker.all(queue.connection)
-    return len(workers) >= 1 and not workers[0].stopped, len(workers)
-
-
-def test_cache():
-    """Test if cache backend is Redis."""
-    if getattr(settings, "CACHES", None):
-        return "RedisCache" in settings.CACHES['default']['BACKEND']
-
-
-def test_cache_server_connection():
-    """Test if we can connect to the cache server."""
-    from django_redis import get_redis_connection
-    return get_redis_connection()
-
-
-##############################
-# Test optional dependencies #
-##############################
-
-
-def test_iso_codes():
-    import gettext
-    languages = (lang[0] for lang in settings.LANGUAGES)
-    if not languages:
-        # There are no UI languages, which is a problem, but we won't complain
-        # about that here.
-        languages = ['af', 'ar', 'fr']
-    return len(gettext.find('iso_639', languages=languages, all=True)) > 0
-
-
-def test_levenshtein():
-    try:
-        import Levenshtein
-        return True
-    except ImportError:
-        return False
-
-
-######################
-# Test optimal setup #
-######################
-
-def test_mysqldb():
-    try:
-        import MySQLdb
-        return True
-    except ImportError:
-        return False
-
-
-def test_db():
-    """Test that we are not using sqlite3 as the django database."""
-    if getattr(settings, "DATABASES", None):
-        return "sqlite" not in settings.DATABASES['default']['ENGINE']
-
-
-def test_session():
-    """Test that session backend is set to cache or cache_db."""
-    return settings.SESSION_ENGINE.split('.')[-1] in ('cache', 'cached_db')
-
-
-def test_debug():
-    return not settings.DEBUG
-
-
-def test_webserver():
-    """Test that webserver is apache."""
-    return ('apache' in sys.modules or
-            '_apache' in sys.modules or
-            'mod_wsgi' in sys.modules)
```

Comment:
Maybe this can be replaced with a check to ensure it is not being run using `runserver` which is something we want to avoid.